packages/sdk/src/protocols/store.ts

```diff
@@ -315,7 +321,8 @@ export class StoreSDK extends BaseProtocolSDK implements IStoreSDK {
 }
 
 export function wakuStore(
+  connectionManager: ConnectionManager,
```

Comment:
nit: let's include it into `init` and maybe make it of a type `ProtocolCreateOptions & ProtocolServices` or something